services/crawler/crawler_test.go

```diff
@@ -0,0 +1,67 @@
+package crawler_test
+
+import (
+	"fmt"
+
+	. "google-scraper/services/crawler"
+	. "google-scraper/constants"
+	. "google-scraper/tests/testing_helpers"
```

Comment:
Prefer to follow the nascent Gin template in which there is one test package:

```
├── tests/
│   └── fixtures/
│   └── http.go
│   └── path.go
```

So then:

```
. "google-scraper/tests/"
```

Not sure if it would clash with Beego though 😓 Also note that this can be done in another PR.

Reply:
I think it should not clash. I will do it another task